tinkernet/runtime/src/lib.rs

```diff
@@ -549,7 +549,8 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
     type OnSystemEvent = ();
     type SelfParaId = parachain_info::Pallet<Runtime>;
-    type DmpQueue = frame_support::traits::EnqueueWithOrigin<MessageQueue, xcm_config::RelayOrigin>;
+    type DmpQueue =
+        frame_support::traits::EnqueueWithOrigin<MessageQueue, xcm_config::RelayAggregate>;
```

Comment:
I don't think this is right.

https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs#L668

Reply:
RelayAggregate is a abstraction of RelayOrigin, this ensures that the custom implementation of process_message written for pallet_ocif_staking does not interfere with the process_message implementation [tinkernet message queue](https://github.com/InvArch/InvArch-Node/blob/94580c6abbcdeb9c2815c5a681e2fd0b30ac526a/tinkernet/runtime/src/xcm_config.rs#L287-L291), [ocif_staking](https://github.com/InvArch/InvArch-Frames/blob/bbf5a7e5c8485590e85f41ae57aa2220893202e9/OCIF/staking/src/primitives.rs#L391-L406)